src/fields.jl

```diff
@@ -176,3 +179,13 @@ for ft in (:CellField, :FaceFieldX, :FaceFieldY, :FaceFieldZ, :EdgeField)
         end
     end
 end
+
+xnodes(c::CellField) = reshape(c.grid.xC, c.grid.Nx, 1, 1)
```

Comment:
What should these be for `FaceFieldsX`? I wasn't sure because of the ambiguity about their size. Are they size `Nx` or size `Nx+1`?

Reply:
Yeah it's kind of weird but we decided to go with `Nx` faces very early in development. Mostly so that all the arrays are of size `Nx*Ny*Nz`.